framework/src/modules/chain/transport/transport.js

```diff
@@ -235,27 +234,10 @@ class Transport {
 		}
 
 		try {
-			const data = await this.blocksModule.loadBlocksDataWS({
-				limit: 34, // 1977100 bytes
-				lastId: query.lastBlockId,
-			});
-
-			_.each(data, block => {
-				if (block.tf_data) {
-					try {
-						block.tf_data = block.tf_data.toString('utf8');
-					} catch (e) {
-						this.logger.error(
-							'Transport->blocks: Failed to convert data field to UTF-8',
-							{
-								block,
-								error: e,
-							},
-						);
-					}
-				}
-			});
-
+			const data = await this.blocksModule.loadBlocksFromLastBlockId(
+				query.lastBlockId,
```

Comment:
I liked the interface where you could pass an object with limit and lastId.
It looks a bit odd now what this `34` refers to?